bin/test-task.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+"use strict";
+
+var util = require("util");
+
+var assert = require("assert"),
+    async = require("async"),
+    fs = require("fs"),
+    tmp = require("tmp"),
+    env = require("require-env");
+
+var worker = require("../lib/worker"),
+    upload = require("../lib/localUploader.js");
+
+var runWorker = function(task, callback) {
+  worker(0, task, upload, function(err) {
+    if (err) {
+      callback(err);
+    }
+
+    return callback(null, arguments);
+  });
+};
+
+var argv = process.argv.slice(2);
+
+assert.equal(1, argv.length, "A file containing the operation json must be provided.");
+
+var task = JSON.parse(fs.readFileSync(argv.shift(), 'utf8'));
```

Comment:
There's nifty Node shorthand for this:

```javascript
var task = require(argv.shift());
```

To better handle relative vs. absolute paths:

```javascript
var task = require(path.resolve(argv.shift()));
```

(I'm also partial to double quotes, or at least consistency within a file, but the ones here can go away.)